easybuild/easyconfigs/n/Ninja/Ninja-1.10.1-GCCcore-10.2.0.eb

```diff
@@ -14,18 +14,44 @@ checksums = ['a6b6f7ac360d4aabd54e299cc1d8fa7b234cd81b9401693da21221c62569a23e']
 
 builddependencies = [
     ('binutils', '2.35'),
+    ('CMake', '3.20.1'),
+    ('scikit-build', '0.15.0'),
```

Comment:
didn't see the other PR

Reply:
see #16024

I don't see any harm in adding an easyconfig for `scikit-build` 0.15.0 with `GCCcore` 10.2-11.3, since i) we're currently not using that anywhere, ii) it's a build-only dep